packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -1879,7 +2073,12 @@ export class Subnet extends Resource implements ISubnet {
       cidrBlock: props.cidrBlock,
       availabilityZone: props.availabilityZone,
       mapPublicIpOnLaunch: props.mapPublicIpOnLaunch,
+      ipv6CidrBlock: props.ipv6CidrBlock,
+      assignIpv6AddressOnCreation: props.assignIpv6AddressOnCreation,
```

Comment:
Set enable_dns64 = true (if protocol is dual stack), will simplify creation of IPv6-only machines on the VPC.  A machine can always use an alternative, e.g. Google DNS64, provided there is a NAT64 route (see other comments), but setting here will make it automatic.

NOTE: You need to have NAT64 routes set up before enabling this.

Also (can be in the future) suggest adding an override when creating custom subnets as there could be some cases where you don't want NAT64 turned on.